### ISSUE TITLE
WS281x leds flickering every firmware after WLED 0.15.0-b3

### ISSUE THREAD

**Issue by Justdigit** [wled/WLED#4211]
### What happened?

Leds are flickering in all kinds of colors and cant acces the webpage.
Its an Esp32 with six channels

### To Reproduce Bug

WLED firmware WLED 0.15.0-b3 works well. All versions after that not.

### Expected Behavior

Responsive webpage and normal color changes

### Install Method

Binary from WLED.me

### What version of WLED?

WLED 0.15.0-b3

### Which microcontroller/board are you seeing the problem on?

ESP32 wroom
GPIO Pins 12,13,14,25,26,27

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by softhack007** [comment]
Can you share a screenshot of your LEDs preferences? 

In case you have less then 300 LEDs per pin, AND more than 4 pins used, WLED uses a different LEDs driver (called "parallelI2S1"). It might be that your problems is related to this new driver.

**Comment by Justdigit** [comment]
> Can you share a screenshot of your LEDs preferences?
> 
> In case you have less then 300 LEDs per pin, AND more than 4 pins used, WLED uses a different LEDs driver (called "parallelI2S1"). It might be that your problems is related to this new driver.

Yes thats true there are "only" 218 leds with four channels. This is because it's a PC case with RGB fans in it.
Is there a solution for it??

![LED-Settings](https://github.com/user-attachments/assets/00a63922-2f97-4f6e-a2b4-75a401b0cb75)

**Comment by jonathh21** [comment]
It might not be related, but I have a matrix of 1024 LEDs.

At low brightness it goes crazy, random pixels and brightness galore.

Higher brightness less prevalent and ultimately bright enough it stops.

(I have separately observed that transitions causes instability so I have disabled it),

**Comment by Justdigit** [comment]
> It might not be related, but I have a matrix of 1024 LEDs.
> 
> At low brightness it goes crazy, random pixels and brightness galore.
> 
> Higher brightness less prevalent and ultimately bright enough it stops.
> 
> (I have separately observed that transitions causes instability so I have disabled it),

Idk if it's related.... 
When I upload a firmware higher than B3 i cant even acces the webpage tp fi in WiFi credentials.

**Comment by lost-hope** [comment]
Just to rule out that the flickering is a hardware issue: Do you two have a levelshifter installed?

**Comment by Justdigit** [comment]
> Just to rule out that the flickering is a hardware issue: Do you two have a levelshifter installed?

No I have not.... Because prior versions just works right. They are small strings of leds.

**Comment by Justdigit** [comment]
> Can you share a screenshot of your LEDs preferences?
> 
> In case you have less then 300 LEDs per pin, AND more than 4 pins used, WLED uses a different LEDs driver (called "parallelI2S1"). It might be that your problems is related to this new driver.

I have posted my config on you request.... Can you do something with it?

**Comment by DedeHai** [comment]
if you reduce the number of outputs to 4, does it work correctly?

**Comment by Justdigit** [comment]
> if you reduce the number of outputs to 4, does it work correctly?

No that doesnt matter. 2 CHannels or six. It keeps flickering

**Comment by DedeHai** [comment]
then it may be related to your hardware. please use recommended wiring, i.e. add a [levelshifter](https://kno.wled.ge/basics/compatible-hardware/#levelshifters) to rule that one out.

**Comment by Justdigit** [comment]
> then it may be related to your hardware. please use recommended wiring, i.e. add a [levelshifter](https://kno.wled.ge/basics/compatible-hardware/#levelshifters) to rule that one out.

No it's definitely not the hardware and doesn't need level shifters. 
Version B3 works like a charm. All version later not. 
The longest cable is from esp to case fan is 20cm.

Only thing I didn't try is a different esp.

**Comment by DedeHai** [comment]
> No it's definitely not the hardware (case Fans and led strip) and doesn't need level shifters.

in 99% of cases, flickering is a hardware problem. I cannot reproduce you issue so please update your hardware.

**Comment by mozzhead164** [comment]
I can also confirm the flickering when I updated to either '0.15.1.beta2_ESP32_Ethernet' or '0.16.0-alpha_ESP32_Ethernet' from '0.15.0-b7_ESP32_Ethernet' today. 

b7 was working perfectly until I updated. 

I am using level shifters on 2 outputs with approx 300 LED's per output (see screenshot below)
The controller is a Dig-Octa which was previously outputting smooth LED animation, have reverted back to b7 for now.

![Image](https://github.com/user-attachments/assets/9a3aa609-b6fd-4cea-aa61-f451bb38a573)

**Comment by willmmiles** [comment]
Possible duplicate of #4389
